### PR TITLE
OSDOCS-9865: Specialized Hardware and Driver Enablement Table

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/OSDOCS-9865

Link to docs preview: https://77405--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html

QE review:
- [X] QE has approved this change.

Additional information: Specialized hardware and driver table in 4.16 release notes
image

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Comment by GroceryBoyJr** [comment]
@bthurber PTAL

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -524,6 +524,21 @@ In the following tables, features are marked with the following statuses:
...
+
+|Hub and spoke cluster support
+|General Availability
+|General Availability
+|General Availability
+
+|Kernel Module Management Operator
+|General Availability
+|General Availability
+|General Availability
```
Comment (qbarrand) [inline release_notes/ocp-4-16-release-notes.adoc:540]:
```suggestion
|Kernel Module Management Operator
|General Availability
|General Availability
|General Availability

|Kernel Module Management Operator - support for Hub and Spoke topologies
|General Availability
|General Availability
|General Availability
```

**Review by cdvultur — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -524,6 +524,21 @@ In the following tables, features are marked with the following statuses:
 |====
 |Feature |4.14 |4.15 |4.16
 
+|Driver Toolkit
+|General Availability
+|General Availability
+|General Availability
+
+|Hub and spoke cluster support
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply release_notes/ocp-4-16-release-notes.adoc]: Done, thank you @cdvultur

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (GroceryBoyJr) [reply release_notes/ocp-4-16-release-notes.adoc]: Done, thank you @qbarrand

File: release_notes/ocp-4-16-release-notes.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply release_notes/ocp-4-16-release-notes.adoc]: @cdvultur I was asked not to change the name of the table. But I have gladly made the other changes you requested.

**Comment by GroceryBoyJr** [comment]
@cdvultur @qbarrand I was unable to add a fourth table column for 4.13! (that would explain why all the other tables only have only 3!) :-/ I will research this issue and get back to you. In the meantime, I'll push it to the release notes with three releases shown so the information is not lost from the release notes.

**Pull request merged**